colibre/auto_plotter/stellar_mass_halo_mass.yml

```diff
@@ -588,3 +588,152 @@ stellar_mass_halo_mass_M200_centrals_30:
+    section: Stellar Mass-Halo Mass (BCGs)
+    show_on_webpage: true
+  observational_data:
+    - filename: GalaxyStellarMassHaloMassBCG/DeMaio2020_10kpc.hdf5
+    - filename: GalaxyStellarMassHaloMassBCG/GoldenMarx2023_10kpc.hdf5
+    - filename: GalaxyStellarMassHaloMassBCG/Huang2022_10kpc.hdf5
+
+stellar_mass_halo_mass_M200_centrals_projected_30:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "masses.mass_200crit"
+    units: Solar_Mass
+    start: 1e13
+    end: 5e15
+  y:
+    quantity: "derived_quantities.projected_stellar_mass_30_kpc"
+    units: Solar_Mass
+    start: 5e10
+    end: 1e13
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 10
+    start:
+      value: 1e13
+      units: Solar_Mass
+    end:
+      value: 5e15
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Halo Mass relation (30 kpc 2D aperture, $M_{200,crit}$, centrals only)
+    caption: Includes only central haloes, and excludes the contribution of satellites in projection, for both the simulated and observed data. The halo masses in observations by Kravstov et al. (2018) and the semi-empirical model by Huang et al. (2021) were converted from the $M_{500}$ and $M_\mathrm{BN98}$ definitions, respectively, to a $M_{200}$ definition (critical overdensity), by assuming an NFW profile and concentration $c=5$. The halo masses from Golden-Marx et al. (2022) were converted from a mean overdensity to critical overdensity (factor 200) definition. None of the halo masses are corrected for (hydrostatic and other) bias. The IMF from observations by Golden-Marx et al. (2022) was converted from Salpeter to Chabrier. Cosmology corrections were not needed. The data that are of intermediate-redshift objects can be used for comparison here because the evolution of BCG stellar cores is observed to be negligible in this regime. The Huang et al. (2021) model is tailored to groups and clusters, and uses aperture definitions for stellar masses, not only for its predictions but also for the observational data used for calibration.
+    section: Stellar Mass-Halo Mass (BCGs)
+    show_on_webpage: true
+  observational_data:
+    - filename: GalaxyStellarMassHaloMassBCG/Zhang2016_30kpc.hdf5
+    - filename: GalaxyStellarMassHaloMassBCG/Lin2017_30kpc.hdf5
+    - filename: GalaxyStellarMassHaloMassBCG/Kravstov2018_30kpc.hdf5
+    - filename: GalaxyStellarMassHaloMassBCG/GoldenMarx2023_30kpc.hdf5
+    - filename: GalaxyStellarMassHaloMassBCG/Huang2022_30kpc.hdf5
```

Comment:
Has the submodule `velociraptor-comparison-data` been updated to include the BCG observational data, which was added recently?

Reply:
Should be done now.